### DISCUSSION TITLE
[Announcement] Coming changes to note templates in v1.0.0+

### DISCUSSION THREAD

**Discussion by windingwind** [windingwind/zotero-better-notes#421] [Announcements]
Hi everyone,

There is a major version update and the note templates face very big changes:

- Some input variables are removed or changed
- All of the built-in APIs under `Zotero.Knowledge4Zotero` are no longer available
- A group of new APIs is under `Zotero.BetterNotes.api`
- Markdown templates are supported with `// @use-markdown`

Currently, it's still in the alpha test. You can find it on the release page.

Best,
windingwind

**Comment by llity** [comment]
新版没有主笔记切换功能

**Reply by windingwind** [reply]
请详细描述具体指的是什么

**Reply by llity** [reply]
主笔记的快捷切换，新版还没有增加吧

**Comment by llity** [comment]
测试版引起tab重复、消失的bug
![image](https://user-images.githubusercontent.com/11771112/233909758-d14da790-ab0b-4f82-b021-aa4a3486d6f2.png)

**Reply by windingwind** [reply]
与本插件无关。已在相关插件中修复。
但请注意，必须将所有使用toolkit包创建阅读器右侧栏的插件都升级到最新版本才能解决问题
